src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -178,6 +178,23 @@ cdef class Buffer:
     def guess_segment_properties(self) -> None:
         hb_buffer_guess_segment_properties(self._hb_buffer)
 
+    def set_message_func(self, func: Callable[[str], bool]) -> None:
+        # XXX I'm passing the python callback as user_data and recovering it
+        # from the C-API func below. Is there a better way?
+        # Also, the callback is only taking a message argument, no buffer,
+        # nor font or its own user_data. Not sure whether/how to pass them on
```

Comment:
We could probably look into this, but do we at all need implementing "message func" in the first place?

The nice thing about the other stuff I put thus far is I use it so I know what things need to be passed to python or not..

Reply:
I don't know if we need to implement it. I was just trying to port the simple debugger that is exemplified in the harfbuzz's original sample.py, but using uharfbuzz.

Reply:
maybe we can leave it out for now, I've never needed to use it myself

Reply:
ok

Reply:
@adrientetar I removed from the example. Shall I remove the set_message_func method altogether from the pyx source file as well?

Reply:
yes, and pxd too